### PR TITLE
Added second way to re-order fields to manual

### PR DESCRIPTION
**Pull request by GithubAnon0000**
Fix: #314

### CODE REVIEW

File: src/editing.md [hunk 1]
```diff
@@ -115,11 +115,18 @@ window.
...
-buttons. To change the order in which the fields appear in this dialog
+buttons.
+
+To change the order in which the fields appear in this dialog
 and the add notes dialog, you can use the reposition button, which asks
 for the numerical position you want the field to have. So if you want to
 change a field to be the new first field, enter "1".
 
+Alternatively you can also drag and drop the field names to re-order them. To do
+that, right-click the field, and while still pressing the right mouse button,
```
[the remark this answers is not in the document]
Comment (brishtibheja) [reply src/editing.md:126]: In other parts of manual, we simply say drag and drop. Or do something like:

```
right-click the field, and drag it to where you want the field to be
```

**Comment by Danika-Dakika** [comment]
This isn't a "right-click" functionality, so I don't think we should document it that way. It works just like a standard drag-and-drop.

**Comment by brishtibheja** [comment]
If you just say "drag it", it's fine with that. A good source to look up
how to document specific things like that is Google's developer
documentation style guide.

2024年12月7日(土) 9:25 GithubAnon0000 ***@***.***>:

File: src/editing.md [hunk 2]
```diff
@@ -115,11 +115,17 @@ window.
...
-buttons. To change the order in which the fields appear in this dialog
+buttons.
+
+To change the order in which the fields appear in this dialog
 and the add notes dialog, you can use the reposition button, which asks
 for the numerical position you want the field to have. So if you want to
 change a field to be the new first field, enter "1".
 
+Alternatively you can also drag and drop the field names to re-order them. To do
+that, click the field, and drag it to the desired position. An indicator will
```
Comment (dae) [inline src/editing.md:126]:
```suggestion
that, use your mouse or finger to drag the field to the desired position. An indicator will
```

**Comment by dae** [comment]
Thank you!

**Pull request merged**